packages/neon-ledger/package.json

```diff
@@ -37,7 +37,9 @@
   },
   "dependencies": {
     "@ledgerhq/hw-transport": "5.51.1",
-    "@types/ledgerhq__hw-transport": "4.21.3"
+    "@types/ledgerhq__hw-transport": "4.21.3",
+    "lodash-es": "4.17.15",
```

Comment:
is this an accidental addition? I dont see it used anywhere

Reply:
`cloneDeep` is import from `lodash-es` and used in `ErrorCode.ts`, i'll ping you there